### PR TITLE
Document data migrations

### PR DESCRIPTION
**Pull request by jamiecobbett**

### CODE REVIEW

**Comment by boffbowsh** [comment]
:+1: but add a note about the separate jenkins job too?

**Comment by jamiecobbett** [comment]
@boffbowsh good idea - rebased.

**Comment by boffbowsh** [comment]
:sparkles: :star2: :star: :+1: :star: :star2: :sparkles:

File: db/data_migration/README.md [hunk 1]
```diff
@@ -0,0 +1,41 @@
+# Data migrations
+
+## What they are and when to use them
+
+Data migrations are generally used for a migration-type activity which:
```
Comment (elliotcm) [inline db/data_migration/README.md]: "migration-type activity" seems a little redundant.

Comment (jamiecobbett) [inline db/data_migration/README.md]: Fair enough, rebased with different words.

**Comment by benlovell** [comment]
:cool:

**Pull request merged**